presto-thrift-connector/pom.xml

```diff
@@ -48,11 +48,6 @@
             <artifactId>bootstrap</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>io.airlift</groupId>
-            <artifactId>json</artifactId>
```

Comment:
This seems unrelated to JMX stats

Reply:
JsonModule was removed, so dependency was removed as well. ClientStats was the only possible module that could require it. In any case, removing it as part of cleaning up extra modules seems fine to me.

Reply:
Remove JsonModule and this dep in a separate commit (could be in same PR), since it's unrelated to removing the stats.